static/app/components/events/evidence.tsx

```diff
@@ -0,0 +1,28 @@
+import EventDataSection from 'sentry/components/events/eventDataSection';
+import KeyValueList from 'sentry/components/events/interfaces/keyValueList';
+import {Event, Group, IssueCategory} from 'sentry/types';
+
+type EvidenceProps = {event: Event; group?: Group};
+
+const Evidence = ({event, group}: EvidenceProps) => {
+  const evidenceDisplay = event.occurrence?.evidenceDisplay;
+
+  if (!evidenceDisplay?.length || group?.issueCategory === IssueCategory.PERFORMANCE) {
```

Comment:
To keep the UI the same for now I'm not rendering this for performance issues, since it already renders this table in `spanEvidence.tsx`. When we figure out how we want to genericize this for performance issues we can revisti

Reply:
Can we actually add this as a comment above the component?

Reply:
Good thinking, added